### PR TITLE
IBX-2886: Help message fields

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-2886
| Versions      | master, 4.2

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/pages/page_block_attributes.md [hunk 1]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
...
+                        help:
+                            text: 'Some exemplary text'
+                            html: true|false
+                            attr:
+                                class: 'class1 class2'
+```
+
+- `help` - defines a help message which is rendered below the field.
+- `help_attr` - sets the HTML attributes for the element which displays the help message.
+- `help_html` - default value: `false`, if set to `true`, the help text is rendered.
```
Comment (ciastektk) [inline docs/content_management/pages/page_block_attributes.md:183]: For `help_html` option I will suggest to use similar description as in the official Symfony documentation e.g.: https://symfony.com/doc/current/reference/forms/types/text.html#help-html

**Review by ciastektk — CHANGES_REQUESTED** [review]

File: docs/content_management/pages/page_block_attributes.md [hunk 2]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
...
+                        help:
+                            text: 'Some exemplary text'
+                            html: true|false
+                            attr:
+                                class: 'class1 class2'
+```
+
+- `help` - defines a help message which is rendered below the field.
+- `help_attr` - sets the HTML attributes for the element which displays the help message.
+- `help_html` - set this option to `true` to not to escape the contents of the `help` option from rendering them in the template.
```
Comment (ciastektk) [inline docs/content_management/pages/page_block_attributes.md:183]:
```suggestion
- `help_html` - set this option to `true` to not escape the contents of the `help` option from rendering them in the template.
```

**Review by ciastektk — APPROVED** [review]

File: docs/content_management/pages/page_block_attributes.md [hunk 3]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
 
 ``` yaml
 [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 9,16) =]] [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 19,26) =]]
-```
\ No newline at end of file
+```
+
+## Help messages for form fields
+
+With the `help`, `help_attr` and `help_html` field options, you can define help messages for fields in the Page block.
```
Comment (DominikaK) [inline docs/content_management/pages/page_block_attributes.md:163]:
```suggestion
With the `help`, `help_attr`, and `help_html` field options, you can define help messages for fields in the Page block.
```

File: docs/content_management/pages/page_block_attributes.md [hunk 4]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
...
+
+```yaml
+ibexa_fieldtype_page:
+    blocks:
+        block_name:
+            attributes:
+                attribute_name:
+                    options:
+                        help:
+                            text: 'Some exemplary text'
```
Comment (DominikaK) [inline docs/content_management/pages/page_block_attributes.md:175]:
```suggestion
                            text: 'Some example text'
```

File: docs/content_management/pages/page_block_attributes.md [hunk 5]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
...
+                            html: true|false
+                            attr:
+                                class: 'class1 class2'
+```
+
+- `help` - defines a help message which is rendered below the field.
+- `help_attr` - sets the HTML attributes for the element which displays the help message.
+- `help_html` - set this option to `true` to not escape the contents of the `help` option from rendering them in the template.
```
Comment (DominikaK) [inline docs/content_management/pages/page_block_attributes.md:183]:
```suggestion
- `help_html` - set this option to `true` to disable escaping the contents of the `help` option when rendering in the template.
```

File: docs/content_management/pages/page_block_attributes.md [hunk 6]
```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
...
+
+### Help message in nested attributes
+
+You can set the options for root or nested attribute, see the exemplary configuration:
```
Comment (DominikaK) [inline docs/content_management/pages/page_block_attributes.md:187]:
```suggestion
You can set the options for root or nested attribute, see the example configuration:
```

**Review by DominikaK — APPROVED** [review]

**Pull request merged**